### PR TITLE
Use ius-archive repo for php 7.2

### PR DESCRIPTION
**Pull request by EMH333**

### CODE REVIEW

File: recipes/packages.rb [hunk 1]
```diff
@@ -25,8 +25,8 @@
   include_recipe 'yum-centos'
   include_recipe 'yum-osuosl'
 
-  # CentOS 7.8 updated ImageMagick which broke installations from ius-archive
-  if enable_ius_archive && node['platform_version'].to_i >= 7
+  # CentOS 7.8 updated ImageMagick which broke installations from ius-archive for older php versions
```
Comment (ramereth) [inline recipes/packages.rb:28]:
```suggestion
  # CentOS 7.8 updated ImageMagick which broke installations from ius-archive for php versions 7.1 and below
```

**Review by ramereth — APPROVED** [review]
Please do a patch bump on `~`

**Comment by EMH333** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v5.5.2.  Have a nice day!

**Pull request merged**